source/content/drupal.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Drupal on Pantheon"
+description: Get started with Drupal on the Pantheon WebOps Platform.
+tags: [site, D8, D9, D10]
+contributors: [populist, edwardangert, whitneymeredith]
+reviewed: "2022-12-09"
+contenttype: [doc]
+categories: [create, plan, config]
+newcms: [drupal]
+audience: [development]
+product: [integrated-composer]
+integration: [--]
+---
+
+The latest version of Drupal is available on Pantheon to all new sites, and is available as an [upgrade path for Drupal 8+ sites](/drupal-migration).
```

Comment:
This is not yet true. We should have a new dashboard entry for Drupal 10 by the end of Feb.

Reply:
Creating a new Drupal 10 site from Terminus will also be available near the end of Feb

Reply:
Changed to `Drupal 9 is available on Pantheon to all new sites, and is available as an [upgrade path for Drupal 8+ sites](/drupal-migration).  For information regarding Drupal 10, see [Drupal 10 on Pantheon](/drupal-10).
`